### DISCUSSION TITLE
Formulas/Calculated Fields on GitHub Issues custom fields

### DISCUSSION THREAD

**Discussion by ianceicys** [community/community#8011] [Projects and Issues]
I am looking for the ability to put formulas across custom fields, say for example the cycle ends on December 3, but the target date for the feature is set to December 2, can I write a formula that calculates the number of days (and allows me to sort by this column) between target date and cycle date. So in the example above, it would show Target (December 2) - Cycle End (December 3) = -1. 

I'd love to tie the custom field calculation into the workflow feature and any feature with a (**negative value**) would get labeled with a **requires_review** tag.

![image](https://user-images.githubusercontent.com/3976990/142939441-e2fbc55a-a7c7-4885-8b87-48c6e80e67aa.png)

Using something like PowerFx for the syntax would be awesome. [https://github.com/microsoft/Power-Fx](https://github.com/microsoft/Power-Fx)

**Comment by vst** [comment]
Duplicate of https://github.com/orgs/community/discussions/6080?
